security.md

```diff
@@ -57,9 +57,9 @@ You would need at least one domain for your website, one for your API, and one f
 
 * A third domain is needed for internal use and back office. This domain would probably be registered anonymously, so it would be a little more difficult to find.
 
-### Use SSL where possible
+### Use SSL/TLS/HTTPS everywhere!
 
-Use SSL anywhere possible. In your website, your API, your back office servers, and if it's not too difficult even between internal services.
+Use SSL anywhere possible. In your website, your API, your back office servers, and if it's not too difficult even between internal services. You can even get away with [free SSL using CloudFlare](https://www.troyhunt.com/how-to-get-your-ssl-for-free-on-shared/) for some time before you need to pay for it.

```

Comment:
You can get free SSL from at least StartSSL, Let's Encrypt and CloudFlare, possibly more. CloudFlare's system requires using them as your DNS server, and possibly forwarding all traffic through them, which you may not want to do (although I've used it in the past, and it is a great service).